examples/nextjs-with-typescript/pages/_document.tsx

```diff
@@ -67,12 +67,7 @@ MyDocument.getInitialProps = async ctx => {
   return {
     ...initialProps,
     // Styles fragment is rendered after the app and page rendering finish.
-    styles: (
-      <React.Fragment>
-        {initialProps.styles}
-        {sheets.getStyleElement()}
-      </React.Fragment>
-    ),
+    styles: [...(initialProps.styles || []), sheets.getStyleElement()],
```

Comment:
What do you think of following the documented approach for styled-component TypeScript? https://github.com/zeit/next.js/blob/ce8956aedac1f8271bf463009394d841073d11b7/examples/with-typescript-styled-components/pages/_document.tsx#L18

```suggestion
    styles: [
      <React.Fragment>
        {initialProps.styles}
        {sheets.getStyleElement()}
      </React.Fragment>
    ],
```

*The definition comes from: https://github.com/zeit/next.js/blob/7cf132f8645caaa3f88144c6adb7ed436c287a7c/packages/next-server/lib/utils.ts#L123.*

Reply:
👍 Ok, I changed to this implementantion, just added a key property because this code generates a react warning.